src/content/governance/index.md

```diff
@@ -0,0 +1,159 @@
+
+The opposite approach, off-chain governance, is where any protocol change decisions happen through an informal process of social discussion, which, if approved, would be implemented in code.
+
+**Ethereum governance happens off-chain** with a wide variety of stakeholders involved in the process.
+
+_Whilst at a protocol level Ethereum governance is off-chain, many use cases built on top of Ethereum, such as DAOs, use on-chain governance._
+
+<ButtonLink to="/dao/">More on DAOs</ButtonLink>
+
+<Divider />
+
+## Who is involved? {#who-is-involved}
+
+There are various stakeholders in the [Ethereum community](/community/), each playing a role in the governance process. Starting from the stakeholders closest to the protocol and zooming out, we have:
+
+- **Protocol Developers** (a.k.a. "Core Developers" ): these people maintain the various Ethereum implementations (e.g. go-ethereum, Nethermind, Besu, Erigon at the execution layer or Prysm, Lighthouse, Nimbus, Teku, Lodestar at the consensus layer). [More on Ethereum clients](/developers/docs/nodes-and-clients/).
+- **EIP Authors**: these people propose changes to the Ethereum protocol, in the form of Ethereum Improvement Proposals (EIPs). [More on EIPs](/eips/).
+- **Miners/Validators**: these people run nodes that can add new blocks to the Ethereum blockchain.
+- **Node Operators**: these people run nodes that propagate blocks and transactions, rejecting any invalid transaction or block that they come across. [More on nodes](/developers/docs/nodes-and-clients/).
+- **Application/Tooling Developers**: these people write applications that are run on the Ethereum blockchain (e.g. DeFi, NFTs, etc.) or build tooling to interact with Ethereum (e.g. wallets, test suites, etc.). [More on dapps](/dapps/).
+- **Application Users**: these people interact with applications on the Ethereum blockchain.
+- **Ether holders**: these people own the ETH token. [More on ETH](/eth/).
+
+_Note: any individual can be part of multiple of these groups (e.g. a protocol developer could champion an EIP, and run a beacon chain validator, and use DeFi applications). For conceptual clarity, it is easiest to distinguish between them, though._
+
+<Divider />
+
+## What is an EIP? {#what-is-an-eip}
+
+One important process used in Ethereum governance is the proposal of **Ethereum Improvement Proposals (EIPs)**. EIPs are standards specifying potential new features or processes for Ethereum. Anyone within the Ethereum community can create an EIP. For example, none of the authors of EIP-721, the EIP that standardized NFTs, have worked directly on Ethereum's protocol development.
+
+<ButtonLink to="/eips/">More on EIPs</ButtonLink>
```

Comment:
This (and others like it) feels more appropriate as an inline lines rather than separate buttons at the end of the section.  Hyperlinked text is pretty standard across the web, so it is unclear why we are doing a sort of "references" at the end of each section.